.gitignore

```diff
@@ -297,3 +297,6 @@ bh_unicode_properties.cache
 # Sublime-github package stores a github token in this file
 # https://packagecontrol.io/packages/sublime-github
 GitHub.sublime-settings
+
+MS Visual Studio
```

Comment:
Is this supposed to be a comment, or is there actually a file called "MS Visual Studio"?